lib/msal-browser/src/interaction_client/NativeInteractionClient.ts

```diff
@@ -366,7 +457,6 @@ export class NativeInteractionClient extends BaseInteractionClient {
                 telemetry: NativeConstants.MATS_TELEMETRY
             },
             extendedExpiryToken: false // Make this configurable?
```

Comment:
Please sync with CCS on this to ensure this is the right thing.  They're adding a lot of coverage to web apps and we don't want the WAM side to break this.

Reply:
Sure. Will do.